src/style.css

```diff
@@ -1 +1,100 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Heavy.ttf') format('truetype');
+  font-weight: bold;
+  font-style: normal;
+}
+
+body {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+}
+
+.search-bar__input {
+  display: block;
+  width: 100%;
+  margin-top: 20px;
+  font-family: Avenir, Arial, Helvetica, sans-serif;
```

Comment:
You have defined a font-family for the `.search-bar__input` class. This is unnecessary as the font-family has already been defined in the body selector which will be inherited by all child elements. You can remove this line.